File: app/src/main/java/com/yujin/fontchecker/model/SizeModel.java
package com.yujin.fontchecker.model;

import android.widget.SeekBar;

import androidx.lifecycle.MutableLiveData;

public class SizeModel {
    public final static int DEFAULT_SIZE = 10;
    public MutableLiveData<Integer> size = new MutableLiveData<>();

    public SizeModel() {
        size.setValue(DEFAULT_SIZE);
    }

    public void onSizeProgressChanged(SeekBar seekBar, int progressValue, boolean fromUser) {
        size.setValue(progressValue);
    }
}

File: app/src/main/java/com/yujin/fontchecker/model/SizeModel.kt
package com.yujin.fontchecker.model

import android.widget.SeekBar
import androidx.lifecycle.MutableLiveData
import com.yujin.fontchecker.util.DEFAULT_TEXT_SIZE

class SizeModel(_size: Int? = DEFAULT_TEXT_SIZE) {
    val size = MutableLiveData<Int>(_size)

    fun onSizeProgressChanged(seekBar: SeekBar, progressValue: Int, fromUser: Boolean) {
        size.value = progressValue
    }
}